frontend/src/editor/chrome/wrapper/app-chrome.tsx

```diff
@@ -55,14 +55,44 @@ export const AppChrome: React.FC<PropsWithChildren> = ({ children }) => {
     />
   );
 
+  const variablePane = (
+    <div className="flex flex-col h-full flex-1 overflow-hidden">
+      <div className="p-3 border-b flex justify-between items-center">
+        <div className="text-sm text-[var(--slate-11)] uppercase tracking-wide font-semibold flex-1">
+          Variables
+        </div>
+        <Button
+          className="m-0"
+          size="xs"
+          variant="text"
+          onClick={() => setIsOpen(false)}
+        >
+          <XIcon className="w-4 h-4" />
+        </Button>
+      </div>
+      {/* {selectedPanel === "errors" && <ErrorsPanel />} */}
+      {selectedPanel === "variables" && (
+        <VariableTable
+          className="flex-1"
+          cellIds={cellIds}
+          variables={variables}
+        />
+      )}
+    </div>
+  );
+
   const helperPane = (
     <Panel
       ref={sidebarRef}
       id="helper"
       key={`helper-${panelLocation}`}
       collapsedSize={0}
       collapsible={true}
-      className="bg-[var(--sage-1)] no-print"
+      className={cn(
+        "bg-white dark:bg-[var(--slate-1)] rounded-lg no-print shadow-mdNeutral editor-panel",
```

Comment:
does `editor-panel` do anything?

Reply:
No, good catch, removed. I was experimenting with that but forgot to remove it